BDArmory/Control/MissileFire.cs

```diff
@@ -5790,8 +5867,8 @@ void UpdateGuardViewScan()
                             }
                         }
                 }
-
-                StartCoroutine(UnderAttackRoutine());
+                FireOCM(false);
+                StartCoroutine(UnderAttackRoutine()); //this seems to be firing all the time, not just when bullets are flying towards craft...?
```

Comment:
I think that's intentional. Under attack includes having missiles incoming, while under fire is only for guns.
Note that under fire doesn't actually require them to be firing, but just targeting us with guns and aiming in roughly our direction. The pilot AI then decides if it's a threat based on the miss distance (line 1477 of BDModulePilotAI.cs).